UDV-Core/examples/Demo.js

```diff
@@ -171,11 +204,21 @@ temporalActiveCtrl.onFinishChange(function(value) {
 var temporalOverlayCtrl = temporalFolder.add(
                                          temporal, 'temporalUsesOverlay'
                                          ).name("Use Overlay").listen();
-
-// Document subwindow
-documentController = datDotGUI.add( documents, 'docBrowserWindowIsActive'
-                                    ).name( "Documents" ).listen();
-documentController.onFinishChange( function(value) { documents.toggleDocBrowserWindow(); });
+//Document uses a folder
+var documentFolder = datDotGUI.addFolder("Documents");
+var docResearch = documentFolder.add( controller.documentResearch, 'windowIsActive').name("Research").listen();
```

Comment:
Respect the (implicit) coding style and the 80 lines wide constraint.
Manually break (newlines) the lines to make the code more readable !